lib/ssh_scan/os/ubuntu.rb

```diff
@@ -3,7 +3,11 @@ module OS
     class Ubuntu
       class Version
         def initialize(version_string)
-          @version_string = version_string
+          if version_string.nil?
+            @version_string = "unknown"
```

Comment:
What if version_string was just nil rather than unknown?

I'm considering removing "unknown" references that have been a convention thus far because they don't follow CPE conventions.